# DEF-D Fysieke Ontwerpopdracht 2 invul template

## Groep informatie 

Aan het einde van deze ontwerpopdracht lever je als groep één gezamenlijk notebook in (als in *PR* op GitHub en de link op Brightspace). Zet jullie groepsnummer, studienummers en namen in onderstaande tabel zodat wij zeker weten dat het groepsvormen goed is gegaan.

| **Groep: 53**     |  |
| :-------------|:-------------|
| Julia van der Maaten | 6505937 |
| Floor van de Schaft | 6246648 |
| Moos Stoop | 6525474 |
|---|---|
| **Planning Groep: 53**     |**Tijdstip / Tijdspanne**  |
|---|---|
| Sanity check 1 en 2 | 11:30 |
| Bouwen eigen condensator | 12:30 |
| Pauze 1| 12:30 - 13:30 |
| Ontwerpen sensor | 13:30 - 14:00 |
| Sensor verbeteren volgens irritatief proces | 14:00 - 15:00 |
| Meten aan sensor| 15:00 - 15:15  |
| Data verwerking en analyse | 15:15 - 16:30 |



## *Opdracht 1*: Sanity check 3

#### Voeg hier een foto van de schets van je plaatcondensator in. Geef daarbij ook duidelijk de waarde van $A$ en $d$ aan.

<img src="platencondensator.png" alt="schets student 1" width="400" style="transform: rotate(-90deg);">

**De twee alluminium platen hebben beide een afmeting van 10 x 10 cm. De afstand tussen de platen is gelijk aan 0.2 cm.**

In [53]:
# Schrijf hier een python functie die de capaciteit van een plaatcondensator uitrekent. Zorg dat de functie
# als input de ontwerpparameters van de vlakke plaat formule accepteert (oppervlakte, afstand en diëlektrische
# constante) en als output de capaciteit van de vlakke plaat condensator met die ontwerpparameters.
# Je gaat deze formule de rest van de dag gebruiken: controleer hem goed!
#epsilon_r is relatieve permittiviteit van een materiaal (standaard 1 voor een vacuum)

import numpy as np

def vlakkeplaat(A, d, epsilon_r = 2.6): #de relatieve permittiviteit bij lucht of vacuum mag deze gelijkgesteld worden aan 1
    epsilon_0 = 8.854187817e-12  # permittiviteit van vacuum in F/m
    C = (epsilon_0 * epsilon_r * A) / d
    return C

#voor onze platencondensator
#gegeven

A_cap = 0.01      #m^2
d_cap = 0.002     #m

antwoord = vlakkeplaat(A_cap, d_cap, epsilon_r = 2.6)*1e12    #pF

# Optioneel: schrijf je functie zodanig dat je arrays als input kan geven en (dus) ook arrays als output.

A_array = np.array([A_cap])
d_array = np.array([d_cap])

def vlakkeplaat(A_array, d_array, epsilon_r = 2.6): #de relatieve permittiviteit bij papier zit tussen de 2.2 en 3
    epsilon_0 = 8.854187817e-12  # permittiviteit van vacuum in F/m
    C = epsilon_0 * epsilon_r * A / d
    print(C)
    return C

print(f"Berekende capaciteit plaatcondensator groep 53 is {antwoord:.1f} pF")



Berekende capaciteit plaatcondensator groep 53 is 115.1 pF


In [55]:
# Gebruik je formule om uit te rekenen wat je verwacht dat de capaciteit van jullie vlakke plaat condensatoren gaat zijn.

A = 0.01
d = 0.002
epsilon_r = np.array([2.0, 4.7, 1.2])

def vlakkeplaat(A, d_array, epsilon_r): #de relatieve permittiviteit bij papier zit tussen de 2.2 en 3
    epsilon_0 = 8.854187817e-12  
    C = epsilon_0 * epsilon_r * A / d_array
    return C

c_values = np.array([])
c_values = vlakkeplaat(A, d, epsilon_r)

print(f"Berekende capaciteit plaatcondensator groep 53 is {c_values[0]*1e12:.1f} pF")
print(f"Berekende capaciteit plaatcondensator groep 53 is {c_values[1]*1e12:.1f} pF")
print(f"Berekende capaciteit plaatcondensator groep 53 is {c_values[2]*1e12:.1f} pF")



Berekende capaciteit plaatcondensator groep 53 is 88.5 pF
Berekende capaciteit plaatcondensator groep 53 is 208.1 pF
Berekende capaciteit plaatcondensator groep 53 is 53.1 pF


#### Voeg hier een foto van je plaatcondensator en van de meting op de scope in. Dus 2 foto's.
<img src="meting.png" alt="schets student 1" width="300">

<img src="opstelling.png" alt="schets student 1" width="300">

### Meting capaciteit
Vul onderstaande tabel in. Bij Sanity check 2 heb je afgeleid hoe je van berekende capaciteit naar berekend voltage gaat en terug. 


| Namen         | Berekende capaciteit   | Berekend Voltage | Gemeten Voltage | Gemeten capaciteit   |
|-------------|-------------:|-------------:|-------------:|-------------:|
| Groep 53| 115.1 pF | 1151 mV | 760.417 mV |76.0 pF |

## *Opdracht 2:* Synthese.

#### Voeg hier voor een foto van je schets(en) van minimaal drie sensoren in.

<img src="spons.png" alt="schets student 1" width="300">

<img src="schuifmaat.png" alt="schets student 1" width="300">

<img src="weegschaal.png" alt="schets student 1" width="300">



## Opdracht 3: Welke sensor gaan jullie maken?

Wij gaan een sensor maken die gewicht meet.

## Opdracht 4: Uitgewerkte schets (afmetingen!) en plot van de ontwerpgrafiek.

#### Voeg hier een foto van jullie uitgewerkte  schets van de sensor die jullie gaan maken in
![uitgewerkte schets](tudelftLogo.png)

In [ ]:
# Plot hier de verwachte ontwerpgrafiek van jullie sensoren.
# Het berekenen van de capiciteit volgens de gemeten spanning

spanning = np.array([]) / 1000   #spanning (V)

def capiciteit(V_out, R, omega, V_in):
   R = 1000                           #ohm
   omega = 628318                     #radialen per seconde
   V_in = 8                           #volt
   C = V_out/R*omega*V_in
   return C

C = np.array([]) *1e12  #gemeten capiciteit (pF)

#Met de capiciteit wordt de d (afstand tussen de platen) berekend

A = 0.01              # oppervlak plaat (m)
epsilon_r = 1.05      # relatieve permittiviteit van de spons

d = np.array([])   #de resultaten, afstand tussen platen (m), wordt opgeslagen in deze array

def spons_indruk(A, C, epsilon_r):
    epsilon_0 = 8.854187817e-12   
    d = epsilon_0 * epsilon_r * A / C
    return d



## Opdracht 5: Invullen tabel LTSpice.

|         | Capaciteit waarde (pF)  | LTSpice berekend output Vpp (mV)|
|-------------|-------------:|-------------:|
| minmale capaciteit| ? |? |
| gemiddelde capaciteit| ? |? |
| maximale capaciteit| ? |? |

## Opdracht 6: Foto's werkende opstelling.

#### Voeg hier de foto's van je sensoren in actie in.
![meting 1](tudelftLogo.png)
![meting 2](tudelftLogo.png)

## Opdracht 7: Plot de kalibratiegrafiek.

In [ ]:
# Plot hier je gemeten kalibratiegrafiek en plot in dezelfde figuur ook je verwachte ontwerpgrafiek.

## Opdracht 8: Vul de tekst aan.

Onze sensor kan nog verbetered worden door de volgende verbeteringen (minimaal 2):

* verbetering 1
* verbetering 2

## Opdracht 9: Uiteindelijke ontwerp en kalibratiegrafiek.

#### Voeg hier een foto van je uiteindelijke sensor in.
![foto sensor](tudelftLogo.png)

In [ ]:
# Plot hier je oorspronkelijke en je uiteindelijke kalibratiegrafieken, fit daar een bijpassende lijn door
# en zet ook de verwachte ontwerpgrafiek erbij

## Opdracht 10: Vul hieronder de conclusie in.

Vul hier de conclusie in

## Opdracht 11: Kopieer hieronder de leerdoelen.
1. Het kunnen gebruiken van de eigenschappen van een elektrische component in een ontwerpproces om een fysische grootheid te meten.
2. Kennis hebben van gedrag van elektrische componenten en meetopstellingen.
3. Onderdelen van een datasheet zoals een kalibratiegrafiek opstellen, gevoeligheid en onzekerheid beschrijven.

kopieer hier de cel(len) waaruit blijkt dat jullie leerdoel 1 beheersen.

kopieer hier de cel(len) waaruit blijkt dat jullie leerdoel 2 beheersen.

kopieer hier de cel(len) waaruit blijkt dat jullie leerdoel 3 beheersen.